niceday-api/index.test.js

```diff
@@ -1,5 +1,80 @@
-describe('Temporary Mock Test Suite', () => {
-  it('Mock Test Case', () => {
-    expect(true).toEqual(true);
+require('isomorphic-fetch');
+
+const NICEDAY_TEST_SERVERPORT = 8080;
+const NICEDAY_TEST_USER_ID = 38527;
+const MOCK_USER_DATA = {
+  id: NICEDAY_TEST_USER_ID,
+  userProfile: {
+    firstName: 'Mr Mock',
```

Comment:
What happend to Dr Bob? Is he dead?

Reply:
May his soul live on forever as nanopublications.

Reply:
Dr Bob got in so much trouble for fraudulently claiming other people's words as his own that he is weary of the concept of mocks. But he may well return if there is sufficient popular demand...